### PR TITLE
Feature/obs scripts

### PR DESCRIPTION
**Pull request by ShastriPaturi**
## Description

Scripts to fetch and convert to IODA v2 for:
1. OSTIA SST
2. FNMOC insitu T&S
3. NSIDC CDR ice concentration
4. ADT

have been added. The datetime variable is also renamed to dateTime.

### Issue(s) addressed
Addresses #206

### CODE REVIEW

File: fnmoc/source_fnmoc.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+else
+    echo $usage
+    exit 1
+fi
+
+d=$out_dir
+pwd=$(pwd)
+mkdir -p $d/${year}/${date}
+cd $d/${year}/${date}
+wget $source/$fn1/$year/${date}00.$fn2.Z --no-check-certificate
```
Comment (jieshunzhu) [inline fnmoc/source_fnmoc.sh:37]: updated $yr with $year
wget doest not complete on Orion, but works on Gaea.

**Review by jieshunzhu — CHANGES_REQUESTED** [review]

File: fnmoc/convert_iodav2_fnmoc.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+set -eu 
+
+usage="usage: $0 yyyymmdd input_path output_path"
+if [[ $# != 3 ]]; then
+    echo $usage
+    exit 1
+fi
+
+date=$1
+year=${dtg:0:4}
```
Comment (jieshunzhu) [inline fnmoc/convert_iodav2_fnmoc.sh]: dtg=>date

File: fnmoc/convert_iodav2_fnmoc.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+input_path=$1
```
Comment (jieshunzhu) [inline fnmoc/convert_iodav2_fnmoc.sh]: $1=>$2

File: fnmoc/convert_iodav2_fnmoc.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                    -i ${input_path}/${date}/${date}'00'.${typ}              \
+                    -o ${output_path}/${date}/insitu_${typ}_fnmoc_${date}.nc \
+                    -d ${date}'12'
+          echo " "
+          echo $typ DONE
+          fi
+      done 
+   fi
+   echo " "
+echo $dtg DONE
```
Comment (jieshunzhu) [inline fnmoc/convert_iodav2_fnmoc.sh]: $dtg=>$date

File: adt/source.adt.nesdis.sh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+pwd=$(pwd)
+
+d=${out_dir}/${date}
+mkdir -p $d
+cd $d
+
+f=${source}/${yr}/rads_adt_*_${yr}${dy}.nc
+echo $f
+file_sfx=nc
+if validate_url $f; then
```
Comment (jieshunzhu) [inline adt/source.adt.nesdis.sh]: The sentence doesnot work.

File: nsidc/source_nsidc_daily.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+mkdir -p $output_path/${year}/${date} && cd $output_path/${year}/${date}
+wget --ftp-user=anonymous ${source}/north/daily/${year}/seaice_conc_daily_nh_${date}_f17_v04r00.nc
+wget --ftp-user=anonymous ${source}/south/daily/${year}/seaice_conc_daily_sh_${date}_f17_v04r00.nc
+    
+# get the latitude, longitude data files
+
+if [ ! -f $output_path/G02202-cdr-ancillary-nh.nc ]; then 
+   wget --ftp-user=anonymous ${source}/ancillary/G02202-cdr-ancillary-nh.nc
+elif [ ! -f $output_path/G02202-cdr-ancillary-sh.nc ]; then
+   wget --ftp-user=anonymous ${source}/ancillary/G02202-cdr-ancillary-nh.nc
```
Comment (jieshunzhu) [inline nsidc/source_nsidc_daily.sh]: nh.nc=>sh.nc

**Review by jieshunzhu — CHANGES_REQUESTED** [review]

File: fnmoc/convert_iodav2_fnmoc.sh (hunk 4 above)
Comment (ShastriPaturi) [reply fnmoc/convert_iodav2_fnmoc.sh]: Corrected

**Review by jieshunzhu — APPROVED** [review]

**Pull request merged**